Integrations/integration-UrlScan.yml

```diff
@@ -0,0 +1,378 @@
+description: Urlscan.io reputation
+detaileddescription: |-
+  This integration checks domain information from the Urlscan.io Database.
+  No authentication is needed.
+  If you want to submit a scan you need an API key.
+  The API key provided by contacting the Urlscan.io directly  by email.
+configuration:
+- display: URL for Urlscan API
+  name: url
+  defaultvalue: https://Urlscan.io/api/v1/
+  type: 0
+  required: false
+- display: API Key (needed only for submitting URLs for scanning)
+  name: apikey
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var sendRequest = function(method, arg, body, headers) {
+
+        url = params.url;
+        if (url[url.length - 1] === '/') {
+            url = url.substring(0, url.length - 1);
+        }
+
+
+        var requestUrl = url + arg;
+        var res = http(
+            requestUrl,
+            {
+                Method: method,
+                Headers: headers,
+                Body: body
+            },
+            params.proxy
+        );
+
+        if ((res.StatusCode < 200 || res.StatusCode >= 300) && res.success === false) {
```

Comment:
Think the logic should be different. if success==false, it should fail regardless of StatusCode, no?